src/services/ledger/index.ts

```diff
@@ -230,26 +230,24 @@ export const LedgerService = (): ILedgerService => {
     }))
 
     try {
-      const [result]: (TxVolume & { _id: null })[] = await wrapAsyncToRunInSpan({
-        namespace: `service.ledger`,
-        fn: async () =>
-          Transaction.aggregate([
-            {
-              $match: {
-                accounts: liabilitiesWalletId,
-                $or: txnTypesObj,
-                $and: [{ timestamp: { $gte: timestamp } }],
-              },
-            },
-            {
-              $group: {
-                _id: null,
-                outgoingSats: { $sum: "$debit" },
-                incomingSats: { $sum: "$credit" },
-              },
-            },
-          ]),
-      })()
+      addEventToCurrentSpan("volume aggregation stats")
```

Comment:
having a start/end event seem redundant - if you want to record a start and end then that is what a span is for. Events are intended for 1-off bits of information.

Reply:
fair. I'll still suggest to merge this one so we have an example on how to do this.